roles/metrics/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+foreman_metrics_url: "https://{{ ansible_fqdn | default('localhost') }}/metrics"
+foreman_pcp_additional_packages: "{{ ['pcp-system-tools'] if ansible_os_family == 'RedHat' else [] }}"
```

Comment:
Do these `foreman_pcp` violate the naming convention rules? Should they be `foreman_metrics_pcp`?

Reply:
Technically yeah.

We do configure `ansible-lint` to accept everything that is `foreman_` prefixed tho, but would be good style to include the role name.
https://github.com/theforeman/foreman-operations-collection/blob/9c78cf04186e24e00b424d5ac4eba03b99f454bb/.ansible-lint#L6